server/cluster/cluster.go

```diff
@@ -1797,6 +1797,11 @@ func (c *RaftCluster) deleteStoreLocked(store *core.StoreInfo) error {
 	return nil
 }
 
+// SetHotPendingInfluenceMetrics sets pending influence in hot scheduler.
```

Comment:
Why need a function?

Reply:
If metrics are in `cluster` or `statistic`, we have no way to collect scheduler data; if mertics is in `scheduler`, we have no way to know when the store goes offline or the pd transfer leader. So I put metrics in `cluster` to clear it in time, and use this function to collect `scheduler` statistics.

Reply:
> we have no way to know when the store goes offline or the pd transfer leader

I didn't the relative logic to handle offline or transfer leader.

Reply:
https://github.com/tikv/pd/blob/e74488f8b121adb751757d071fe3b71993c7b5b9/server/cluster/coordinator.go#L612